src/clarity-angular/popover/dropdown/dropdown.ts

```diff
@@ -56,6 +61,11 @@ export class Dropdown {
     constructor(public elementRef: ElementRef, @SkipSelf() @Optional() public parent: Dropdown,
                 public ifOpenService: IfOpenService) {
         this.anchor = elementRef.nativeElement;
+
+        // Subscribe and broadcast openChanges from the service
+        ifOpenService.openChange.takeUntil(this.ngUnsubscribe).subscribe(change => {
```

Comment:
I remember @youdz and @jeeyun deciding not to use the takeUntil pattern. @jeeyun please correct me if I am wrong.